WordPress/gutenberg
# Autocomplete: Fix Voiceover not announcing suggestions

## What?


Fixes #47767

## Why?


Stupid Voiceover still doesn't support `aria-activedescendant`. Bugs going back for years, wish Apple would fix this.

Inspired by: https://react-spectrum.adobe.com/blog/building-a-combobox.html

## How?


- Adds a `speak` for IOS and Mac only.
- Allows autocomplete to take a text-only label.
- Adds a package to determine OS.

## Testing Instructions





1. On Mac, enable Voiceover.
2. Open a post or page.
3. Type a title.
4. Press Enter to create your first block.
5. Press slash "/" and type "tab" or another search phrase.
6. Use Up and Down Arrow keys and notice how Voiceover announces each block in an `aria-live` region.
7. Try the same on Windows and notice how there is no screen reader announcement via `aria-live`.

### Testing Instructions for Keyboard


## Screenshots or screencast
